rai/node/bootstrap.cpp

```diff
@@ -1124,6 +1110,15 @@ void rai::bootstrap_initiator::notify_listeners (bool in_progress_a)
 	}
 }
 
+void rai::bootstrap_initiator::process_fork (MDB_txn * transaction, std::shared_ptr<rai::block> block_a)
+{
+	std::unique_lock<std::mutex> lock (mutex);
+	if (attempt != nullptr)
```

Comment:
Does it really make sense to compare a `shared_ptr` with `nullptr`?  Maybe `if (attempt)` makes more sense?

Reply:
Thanks, old habits die hard. This is valid, but I agree it's not idiomatic at all.